manim/_config/main_utils.py

```diff
@@ -410,6 +412,14 @@ def _parse_args_no_subcmd(args):
         "section of the config file to define the output folder structure",
     )
 
+    # Overrides default false streaming so streaming can happen
+    # ? Should we really screw over anyone who uses this with the other commands
```

Comment:
I don't understand what you mean here, could you explain?

Reply:
That one I think I'll remove. That was a note to self in case any other commands collided with the livestreaming one causing absolute chaos.

Reply:
That one can go. It was a note to self in case any other commands collided with the livestreaming one. But by running it from `__main__` instead that reminder is now obsolete